libgcc/config.host

```diff
@@ -407,12 +407,12 @@ amdgcn*-*-amdhsa)
 	tmake_file="$tmake_file gcn/t-amdgcn"
 	extra_parts="crt0.o"
 	;;
-arc[eb]*-*-elf*)
+arc-*-elf* | arceb-*-elf*)
 	tmake_file="arc/t-arc"
 	extra_parts="crti.o crtn.o crtend.o crtbegin.o crtendS.o crtbeginS.o"
 	extra_parts="$extra_parts crttls.o"
 	;;
-arc[eb]*-*-linux*)
+arc*-*-linux* | arceb-*-linux*)
```

Comment:
maybe it should be: `arc-*-linux*`

Reply:
Right, spello :) Will fix and push again.